src/sentry/monitors/logic/mark_failed.py

```diff
@@ -94,7 +96,14 @@ def mark_failed(
     monitor_env.refresh_from_db()
 
     # Create incidents + issues
-    if failure_issue_threshold:
+    use_issue_platform = False
+    try:
+        organization = Organization.objects.get(id=monitor_env.monitor.organization_id)
```

Comment:
Is it possible to avoid this lookup? Do we not have this somewhere already? Can we use the get from cache method or whatever that is?

Reply:
oh hmm. i mean we were already doing it before and it was fine. it's not in flagr. let me see